docker-compose/production/django/Dockerfile

```diff
@@ -3,39 +3,52 @@
 # This Dockerfile is not currently used or maintained
 # but may be useful in setting up your own adserver instance.
 
-FROM python:3.8-alpine
+FROM ubuntu:20.04
```

Comment:
😍 

Should we update the comment above this? Should we actually just remove this file if we aren't using or maintaining it?

Reply:
I'd be OK removing it since we don't use it at all, but perhaps providing a reference is a good thing. We do provide a bit of a reference with the development one though. Mostly, I'm just trying to help any community using the server as much as possible.

Reply:
Yea, I think keeping the one we're actively using is the best approach. Agreed the dev one is a good reference. 

I'm 👍 on simplifying the code & env stuff here (eg. `.envs/local/django.sample`), as you mentioned as well, to make it a bit cleaner 👍

Reply:
@mattishaden, we are thinking of removing the production Dockerfile and just having the dev one as a reference since we are actively using and maintaining that one. This would mean you would need to maintain your own production Dockerfile. We could ensure that the dev Dockerfile is up-to-date, working, and well documented/commented. Does that seem OK?

Reply:
Absolutely! Just having a working reference is fine with us. No errors encountered switching from Alpine to Ubuntu and the container size is acceptable.

Reply:
Sounds good. I'll rework this PR to remove the production docker-compose examples and just use the dev Dockerfile as a reference. It will be have ways to disable the analyzer and keep the image small.